crates/sqlbuiltins/src/functions/scalars/postgres.rs

```diff
@@ -475,3 +476,36 @@ impl BuiltinScalarUDF for PgArrayToString {
         FunctionNamespace::Required("pg_catalog")
     }
 }
+
+/// `pg_catalog.version()` implementation.
+///
+/// This provides more informatation that just the 'server_version' session
+/// variable, and includes things like the build triple.
+///
+/// This uses a spoofed version (and so does not match the `version()` function)
+/// since many postgres tools, including sqlalchemy, will check the version
+/// against hard coded values.
+#[derive(Clone, Copy, Debug)]
+pub struct PgVersion;
+
+impl ConstBuiltinFunction for PgVersion {
+    const NAME: &'static str = "version";
+    const DESCRIPTION: &'static str = "Returns the spoofed postgres version of the database";
+    const EXAMPLE: &'static str = "pg_catalog.version()";
+    const FUNCTION_TYPE: FunctionType = FunctionType::Scalar;
+    fn signature(&self) -> Option<Signature> {
+        Some(Signature::exact(vec![], Volatility::Stable))
+    }
+}
+
+impl BuiltinScalarUDF for PgVersion {
+    fn as_expr(&self, _: Vec<Expr>) -> Expr {
+        Expr::Literal(ScalarValue::Utf8(Some(
+            server_version_with_build_info().to_string(),
+        )))
+    }
```

Comment:
@universalmind303 I was originally trying to use `session_var("server_version")` here, but was getting a "Unknown variable: server_version" error.

Using server_version here would be incorrect anyways, but where would I look to enable using it with `session_var`?